### ISSUE TITLE
Adding more model in model training  for better prediction

### ISSUE THREAD

**Issue by Prasadayus** [BamaCharanChhandogi/Diabetes-Prediction#41]
We can more model like xgboost , cat boost and lightgbm model for better accuracy in .ipynb file.
The code can be added here in model training:
![image](https://github.com/BamaCharanChhandogi/Diabetes-Prediction/assets/129419372/2a72b778-6e64-45fb-832e-f4d4537160e5)

**Comment by BamaCharanChhandogi** [comment]
Thank you for creating this issue! 🎉 We'll look into it as soon as possible. In the meantime, please make sure to provide all the necessary details and context. If you have any questions or additional information, feel free to add them here. Your contributions are highly appreciated! 😊

You can also check our [CONTRIBUTING.md](https://github.com/BamaCharanChhandogi/Diabetes-Prediction/blob/main/CONTRIBUTING.md) for guidelines on contributing to this project.

**Comment by Prasadayus** [comment]
@BamaCharanChhandogi  If possible then plz assign me this issue.

**Comment by Sechana28** [comment]
please assign me this
